index.html

```diff
@@ -2434,6 +2434,59 @@ <h3>
           </p>
         </aside>
       </section>
+      <section class="informative">
+        <h3>
+          Web Apps with scope conflicts
+        </h3>
+        <p>
+          It is possible for two web apps to be installed where there are
+          conflicts between the scope. These conflicts come with disadvantages
+          outlined below and they are generally not recommended.
+        </p>
```

Comment:
Agreed. Adding "_with_ the same":

```suggestion
        <p>
          Because scopes are based on URL matching, it is possible for a developer to create
          multiple web applications with the same, overlapping, or nested scopes. Doing so creates
          several issues (detailed below) and is thus considered bad practice.
        </p>
```